lib_collections/models.py

```diff
@@ -253,6 +514,51 @@ def __init__(self, *args, **kwargs):
     )
     unit_contact = models.BooleanField(default=False)
 
+    # Collection Panel Fields
+    # tab in the admin interface is called 'Collection'
+    digital_collection = models.BooleanField(
+        default=False, help_text='Is this a Digital Collection?'
+    )
+    hex_regex = RegexValidator(
+        regex='^#[a-zA-Z0-9]{6}$',
+        message='Please enter a hex color, e.g. #012043'
+    )
+    branding_color = models.CharField(
+        validators=[hex_regex], max_length=7, blank=True
+    )
+    google_font_link = models.URLField(
+        blank=True, help_text='Google fonts link to embed in the header'
+    )
+    font_family = models.CharField(
+        max_length=100,
+        blank=True,
+        help_text='CSS font-family value, e.g. \'Roboto\', sans-serif'
+    )
+    highlighted_records = models.URLField(
+        blank=True,
+        help_text=(
+            '''URL for browse index in IIIF to display
+            results on collection parent page'''
+        )
+    )
+    highlighted = models.URLField(
```

Comment:
This highlighted element is causing an issue with save draft

Reply:
Good catch! I imagine it's because of the triple `'''`.

Reply:
Ugh; yeah, I was talking about this with Kathy.  That was supposed to get deleted, but it looks like I must have accidentally undeleted it.  Anyway, re-deleting it...